### PR TITLE
doc: add busy topic with RST elements

### PR DESCRIPTION
**Pull request by b-gent**
Updating the cheatsheet to create a busy topic.

### CODE REVIEW

**Comment by b-gent** [comment]
I went through a lot of topics and this I found so far.

Two elements that cannot be inserted here and need to be checked in other topics is ``.. glossary::`` and ``.. toctree ::`` .

**Review by greg-fer — APPROVED** [review]

**Review by peknis — APPROVED** [review]

**Comment by umapraseeda** [comment]
Don't we have to add numbered heading?

File: doc/nrf/templates/cheat_sheet.rst [hunk 1]
```diff
@@ -403,9 +481,20 @@ To link to doxygen macros, enums or functions use:
 * :c:macro:`BT_HIDS_INFORMATION_LEN`
 * :c:func:`bt_hids_init`
```
Comment (umapraseeda) [inline doc/nrf/templates/cheat_sheet.rst:482]: How about the rest? 
* Structure: ``:c:struct:``
* Type: ``:c:type:``
* Enum (the list): ``:c:enum:``
* Enumerator (an item): ``:c:enumerator:``
* Structure member: ``:c:member:``

File: doc/nrf/templates/cheat_sheet.rst [hunk 2]
```diff
@@ -211,6 +255,15 @@ Use :ref: to link to IDs and :doc: to link to files.
...
 
+Notes and tips
+==============
+
+.. note::
+   This is a note.
+
+.. tip::
+   This is a tip.
+
```
Comment (umapraseeda) [inline doc/nrf/templates/cheat_sheet.rst:266]: We have .. important::, .. warning::

**Review by peknis — CHANGES_REQUESTED** [review]

**Comment by b-gent** [comment]
> One more missing element: .. toggle:: (example: https://developer.nordicsemi.com/nRF_Connect_SDK_dev/doc/PR-11089/nrf/software_maturity.html#nrf5340-audio)

toggle added

**Review by gmarull — APPROVED** [review]

**Pull request merged**